gaeguli/pipeline.c

```diff
@@ -273,6 +281,80 @@ gaeguli_init_once (void)
   gst_init (NULL, NULL);
 }
 
+static void
+_print_caps (GstCaps * caps)
+{
+  guint j, size = 0;
+
+  size = gst_caps_get_size (caps);
+  for (j = 0; j < size; ++j) {
+    GstStructure *s = gst_caps_get_structure (caps, j);
+    g_autofree gchar *caps_str;
+    caps_str = gst_structure_to_string (s);
+    g_debug ("caps %s", caps_str);
+  }
+}
```

Comment:
`gst_caps_to_string ()` will do this for you.

Reply:
Thanks.